docs/wiki/introduction/sql.md

```diff
@@ -269,7 +269,35 @@ The following trig functions: `sin`, `cos`, `tan`, `cot`, `asin`, `acos`, `atan`
 
 #### String functions
 
-String parsing functions are always helpful. Some are helpful within subqueries, so they make sense as local-additions:
+String functions are always helpful. Some are helpful within
+subqueries, so they make sense as local-additions:
+
+- `concat(ARG1, ARG2, ARG3...)`: Concatenate arguments, ignoring nulls.
+
+    <details>
+    <summary>Concat function example:</summary>
+    <p>
+
+      osquery> .mode line
+
+      osquery> select concat('hello', NULL, ' ', 'world');
+      concat('hello', NULL, ' ', 'world') = hello world
+    </p>
+    </details>
+
+
+- `concat_ws(SEPARATOR, ARG1, ARG2, ARG3...)`: Concatenate arguments, ignoring nulls, and interleaved with `SEPARATOR`.
```

Comment:
Perhaps `join`? That would be consistent with Python and Go.

For me `ws` has a connotation of http://www.cplusplus.com/reference/istream/ws/

Reply:
I had the same thought, but then I remembered `join` has other SQL meaning, and it felt worse. So, postgres syntax it was

Reply:
Ha yeah forgot about `join` hehe

Postgres syntax sounds good.